packages/wouter/src/index.js

```diff
@@ -319,11 +321,16 @@ export const Redirect = (props) => {
   const { to, href = to } = props;
   const [, navigate] = useLocation();
   const redirect = useEvent(() => navigate(to || href, props));
+  const { ssrContext } = useRouter();
```

Comment:
There is an internal helper hook `useLocationFromRouter` that you can use combined with `useRouter()` to avoid extra context calls.

Reply:
Not sure I understand this suggestion, there's no need to use location (the result of `useLocationFromRouter`) in this instance.